techniques/failures/F47.html

```diff
@@ -1,3 +1,8 @@
+---
+obsoleteSince: 20
+obsoleteMessage: |
+  The <a href="https://html.spec.whatwg.org/multipage/obsolete.html#blink"><code>blink</code> element is obsolete in the HTML Living Standard</a>.
```

Comment:
similar to `text-decoration: blink` - safari, firefox and chromium browsers also do no longer support the "blinking" behavior.  Should "and current has no modern browser support" be added here as well?

Reply:
'tis done. thanks.